contributing/communications.md

```diff
@@ -7,9 +7,134 @@ permalink: /contributing/communications/
 
 # Communications
 
-**:warning: This page is not complete yet :warning:**
+Hello, so you you're interested in helping out with our communications? We're pretty pleased about that. This guide covers general writing style as well as everything you need to start talking to our users on [GitHub](#github) and [Twitter](#twitter). We also cover how to [respond to Code of Conduct issues](#code-of-conduct).
```

Comment:
"Hello, so you you're interested in helping out" should be "Hello, so you're interested in helping out"

extra 'you' that isn't needed